### PR TITLE
N32: Add Terrain Generator and Bloom3D

### PR DESCRIPTION
**Pull request by kettle11**
For #979

### CODE REVIEW

File: content/news/032/index.md [hunk 1]
```diff
@@ -252,6 +252,24 @@ March's full weekly devlogs: "This Week In Veloren...":
...
+### [Terrain Generator][terrain-generator]
+
+![A screenshot looking down on mountainous terrain and an ocean](terrain_generator.jpg)
+
+"Terrain Generator" is a website made by [@kettlecorn][kettlecorn_twitter] that
+generates procedural island landscapes. It uses WebAssembly SIMD and multithreading
+via WebWorkers to speed up generation.
+
+The terrain is generated with multi-octave simplex noise using the new
+simplex-noise crate [`clatter`](https://github.com/Ralith/clatter).
```
Comment (AngelOnFira) [inline content/news/032/index.md:264]: Links to the bottom please :)

File: content/news/032/index.md [hunk 2]
```diff
@@ -252,6 +252,24 @@ March's full weekly devlogs: "This Week In Veloren...":
...
+
+Terrain Generator's code is open-soruced on [GitHub](terrain-generator-github).
```
Comment (AngelOnFira) [inline content/news/032/index.md:266]:
```suggestion
Terrain Generator's code is open-soruced on [GitHub][terrain-generator-github].
```

File: content/news/032/index.md [hunk 3]
```diff
@@ -295,6 +313,20 @@ series.
 
 ## Tooling Updates
 
+### [Bloom3D][bloom3d]
+
+![A screenshot of Bloom3D's interface and a simple low-polygon building.](bloom3d.jpg)
+
+[Bloom3D][bloom3d] is an extremely minimalist in-browser 3D modeling tool made
+by [@kettlecorn](kettlecorn_twitter)
```
Comment (AngelOnFira) [inline content/news/032/index.md:321]:
```suggestion
by [@kettlecorn][kettlecorn_twitter]
```

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
A few change :+1:

**Review by AngelOnFira — APPROVED** [review]

File: content/news/032/index.md [hunk 4]
```diff
@@ -318,6 +318,24 @@ fun and test your skills.
 
 [rust-nonogram]: https://github.com/henryksloan/rust-nonogram
 
+### [Terrain Generator][terrain-generator]
+
+![A screenshot looking down on mountainous terrain and an ocean][terrain_generator.jpg]
```
Comment (17cupsofcoffee) [inline content/news/032/index.md:323]:
```suggestion
![A screenshot looking down on mountainous terrain and an ocean](terrain_generator.jpg)
```

File: content/news/032/index.md [hunk 5]
```diff
@@ -318,6 +318,24 @@ fun and test your skills.
...
+![A screenshot looking down on mountainous terrain and an ocean](terrain_generator.jpg)
+
+"Terrain Generator" is a website made by [@kettlecorn][kettlecorn_twitter] that
+generates procedural island landscapes. It uses WebAssembly SIMD and multithreading
+via WebWorkers to speed up generation.
+
+The terrain is generated with multi-octave simplex noise using the new
+simplex-noise crate [`clatter`][clatter].
+
+Terrain Generator's code is open-soruced on [GitHub][terrain-generator-github].
```
Comment (17cupsofcoffee) [inline content/news/032/index.md:332]:
```suggestion
Terrain Generator's code is open-sourced on [GitHub][terrain-generator-github].
```

**Review by 17cupsofcoffee — APPROVED** [review]

**Pull request merged**